save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserPermissions.kt

```diff
@@ -0,0 +1,13 @@
+package com.saveourtool.save.info
+
+import kotlinx.serialization.Serializable
+
+/**
+ * @property isPermittedCreateContest permission for create contests in organizations
+ * @property isPermittedToBulkUpload permission for upload COSV files in organizations
+ */
+@Serializable
+data class UserPermissions(
+    val isPermittedCreateContest: Map<String, Boolean> = emptyMap(),
```

Comment:
suggestion to split it to two classes:
1. ```kotlin
    @Serializable
     data class UserPermissions(
         canCreateContest: Boolean,
         canDoBulkUpload: Boolean,
     )
    ```
2. ```kotlin
    data class AllUserPermissions(
        val inOrganization: Map<String, UserPermissions>,
    )
    ```

It allows to extend UserPermissions in single place using only Organization and allows to add another permissions per project\vulnerability\contest and etc

Reply:
Ok, agree. Fixed it